src/PowerTools/source.extension.vsixmanifest

```diff
@@ -1,9 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Vsix xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Version="1.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2010">
-  <Identifier Id="2b119c79-9836-46e2-b5ed-eb766cebbf7c">
-    <Name>Entity Framework Power Tools Beta 5</Name>
-    <Author>Microsoft</Author>
-    <Version>0.9.0.0</Version>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
```

Comment:
@divega It is quite confusing - maybe it should have been called VSIX2+ or VSIX 2.5 😄